cmd/notify-mailer/main.go

```diff
@@ -456,10 +457,26 @@ func main() {
 	defer log.AuditPanic()
 
 	// Setup database client.
-	dbURL, err := cfg.NotifyMailer.DB.URL()
+	dsn, err := cfg.NotifyMailer.DB.URL()
 	cmd.FailOnError(err, "Couldn't load DB URL")
 
-	dbMap, err := sa.NewDbMap(dbURL, sa.DbSettings{MaxOpenConns: 10})
+	conf, err := mysql.ParseDSN(dsn)
+	cmd.FailOnError(err, "Couldn't parse DB URL as DSN")
+
+	// Transaction isolation level READ UNCOMMITTED trades consistency for
+	// performance.
+	conf.Params = map[string]string{
+		"tx_isolation": "'READ-UNCOMMITTED'",
+	}
```

Comment:
Same comment here as on id-exporter: gotta make sure that we aren't dropping any conf.Params that came from the original dbURL.

Reply:
Oh gosh, I totally forgot to change this to an assignment rather than a literal. Copypasta 🤦🏻‍♀️